plotly/tools.py

```diff
@@ -48,11 +48,15 @@ def warning_on_one_line(message, category, filename, lineno, file=None, line=Non
 # this sets both the DEFAULTS and the TYPES for these items
 _FILE_CONTENT = {CREDENTIALS_FILE: {'username': '',
                                     'api_key': '',
+                                    'proxy_username': '',
+                                    'proxy_passwd': '',
+                                    'api_key': '',
                                     'stream_ids': []},
                  CONFIG_FILE: {'plotly_domain': 'https://plot.ly',
                                'plotly_streaming_domain': 'stream.plot.ly',
                                'plotly_api_domain': 'https://api.plot.ly',
-                               'plotly_ssl_verification': True}}
+                               'plotly_ssl_verification': True,
+                               'plotly_proxy_authorization': False}}
```

Comment:
is there some terminology we can use to specify which kind of proxy this is? I can imagine that some folks might stumble upon this because they have a company proxy which they need to authenticate past just to get outside of their network. like, `plotly_reverse_proxy_authorization`?